qt/aqt/main.py

```diff
@@ -1430,11 +1430,11 @@ def setupSystemSpecific(self) -> None:
             self.maybeHideAccelerators()
             self.hideStatusTips()
         elif isWin:
-            # make sure ctypes is bundled
             from ctypes import windll, wintypes  # type: ignore
 
-            _dummy = windll
-            _dummy = wintypes
+            # make sure ctypes is bundled
```

Comment:
why did you move the comment?

Reply:
I thought the `bundled` thing was happening because of the dummy variables being assignment. I moving it back.